src/zzz_od/application/inventory_scan/special_scan/special_scan_app.py

```
@@ -0,0 +1,303 @@
+        self._img_portrait = cv2_utils.crop_image_only(screen, area_portrait.rect)
+        self._img_name = cv2_utils.crop_image_only(screen, area_name.rect)
+        self._img_level = cv2_utils.crop_image_only(screen, area_level.rect)
+
+        self.round_by_goto_screen(screen_name='代理人-技能')# 导航到技能界面
+        time.sleep(0.3)# 等待技能界面加载完成
+        #return self.round_fail('已导航到特定代理人的技能界面')
+        
+        #技能信息截图
+        screen = self.screenshot()
+        if screen is None:
+            log.error("截图失败，无法获取屏幕画面")
```

Ruff (0.15.7)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)